### PR TITLE
Remove flash based xtypes

### PR DESCRIPTION
**Pull request by Jako**
### What does it do?
Remove flash based xtypes

### Why is it needed?
It could be possible, that sites running any Flash will be actively blocked by Chrome.

### How to test
The manager works with this changes and the xtypes removed are not used in the core.

### Related issue(s)/PR(s)
https://github.com/modxcms/revolution/issues/13199#issuecomment-717908136

### CODE REVIEW

**Comment by Jako** [comment]
This PR could be backported/copied to 2.x

**Comment by JoshuaLuckers** [comment]
What if an extra relies on these?

**Comment by Jako** [comment]
Then the extras have a different issue, because browser based flash is EOL. 

Maybe we could add the removed parts to an extra file.

**Comment by Ruslan-Aleev** [comment]
Maybe in the future it makes sense to transfer/remove other components that are not used?
If there are many of them, then by removing them we will lighten the size of the distribution, we can avoid potential errors in the future (with flash, for example) and even update extJS components separately.

**Comment by Jako** [comment]
http://swfobject.googlecode.com/svn/trunk/swfobject/expressInstall.swf and http://yui.yahooapis.com/2.8.2/build/charts/assets/charts.swf don't exist anymore. So the Ext.chart things could be removed. They can't be used anymore and should currently throw an error.

**Comment by Jako** [comment]
The FileApi code has a fallback to flash, but the referenced files are not available anymore. It could be disabled quite easy.

**Review by JoshuaLuckers — APPROVED** [review]

File: manager/assets/modext/util/utilities.js [hunk 1]
```diff
@@ -421,28 +421,8 @@ Ext.applyIf(Ext.form.Field,{
...
-                }
-                document.getElementById(flashcopier).innerHTML = '';
-                var divinfo = '<embed src="' + MODx.config.manager_url
-                    + 'assets/modext/_clipboard.swf" FlashVars="clipboard='
-                    + MODx.util.Clipboard.escape(text)
-                    + '" width="0" height="0" type="application/x-shockwave-flash"></embed>';
-                document.getElementById(flashcopier).innerHTML = divinfo;
-            }
+        copy: function(text){
+            console.log('MODx.util.Clipboard was deactivated!');
```
Comment (JoshuaLuckers) [inline manager/assets/modext/util/utilities.js]: I suggest removing this component completely. If there are good reasons to keep it we should log an error instead.

**Review by JoshuaLuckers — CHANGES_REQUESTED** [review]

File: manager/assets/modext/util/utilities.js (hunk 1 above)
Comment (Jako) [reply manager/assets/modext/util/utilities.js]: A github code search for MODx.util.Clipboard shows 459 results, most in the utilities.js file or in some cache files. So I think that the component is no longer in use.

**Review by Ibochkarev — APPROVED** [review]

**Review by JoshuaLuckers — APPROVED** [review]

**Pull request merged**